sdk/python/express_relay/svm/limo_client.py

```diff
@@ -143,7 +143,7 @@ async def get_init_if_needed_wsol_create_and_close_ixs(
 
         create_ixs = []
         close_ixs = []
-        if ata_info.value is None:
+        if ata_info.value is None or len(ata_info.value.data)==0:
```

Comment:
This fixes a bug where the account can exist but not be an ATA, for example if someone sends sol to the account